src/server/constants.py

```diff
@@ -67,3 +67,5 @@ class QUERIES:
                         LIMIT 1"""
     WALLET_BUY_ADD_BTC: Query = "UPDATE users SET wallet_btc = wallet_btc + %s WHERE uuid=%s"
     WALLET_BUY_SUBTRACT_USD: Query = "UPDATE users SET wallet_usd = wallet_usd - %s WHERE uuid=%s"
+    WALLET_SELL_SUBTRACT_BTC: Query = "UPDATE users SET wallet_btc = wallet_btc - %s WHERE uuid=%s"
+    WALLET_SELL_ADD_USD: Query = "UPDATE users SET wallet_usd = wallet_usd + %s WHERE uuid=%s"
```

Comment:
`WALLET_BUY_SUBTRACT_USD` and `WALLET_SELL_ADD_USD` are the same as `WALLET_WITHDRAW` and `WALLET_DEPOSIT` respectively. Maybe it's better to use those instead?

Reply:
This should be dicussed with @vis4rd

Reply:
For sure buy and sell operations should be separate from withdraw and deposit, even though for database it's virtually the same query. I think that for now let's keep it separate for consistency, we can worry about duplication once it hits us with some bug.